catalogs/views.py

```diff
@@ -1,81 +1,49 @@
 from pathlib import Path
 
 from rest_framework import generics, status
-from rest_framework.decorators import api_view
+from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
+from rest_framework.request import Request
 
 from .catalogio import CatalogTools as Tools
 from .models import Catalog
 from .serializers import CatalogListSerializer
 
 
 class CatalogListView(generics.ListCreateAPIView):
-    """
-    Use for read-write endpoints to represent a collection of model instances.
+    """Use for read-write endpoints to represent a collection of model instances.
     Provides get and post method handlers.
     """
-
     queryset = Catalog.objects.all().order_by("pk")
+    permission_classes = [AllowAny, ]
```

Comment:
`AllowAny` is open permissions, but let me know if this should be read-only.

Reply:
Question:  Is this the temporary work around that we discussed until we follow up with more exacting permissions?

Reply:
Exactly. I wanted to point it out here just in case someone did have a thought before we start with this.